test/library/packages/UnitTest/PopulateTests_PRIM/TestCountClass.future

```diff
@@ -0,0 +1 @@
+Feature: Class methods should be considered as tests
```

Comment:
This should be `feature request:` instead.  Also, it would be good to have an issue number associated with it

Reply:
Thanks, I have updated it.